scripts/rewrite-decl-statements-for-closure-test.js

```diff
@@ -22,7 +22,7 @@
  *  * Rewrite import Foo from './foo' to const Foo = goog.require('mdc.foo')
  *  * Rewrite import {foo, bar} from './util' to const {foo, bar} = goog.require('mdc.util')
  *
- *  
```

Comment:
Running `npm run lint` resulted in an error pointing to this line. I noticed that on TravisCI, and so I made this fix to let the validation process continue.